pyrene/src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,303 @@
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+
+import ReactDatePickerWrapper from './ReactDatePickerWrapper/ReactDatePickerWrapper';
+import DateTimeInput from './DateTimeInput/DateTimeInput';
+
+import {
+  getFutureDate, customDateFormat, convertToUTCtime,
+  convertToZoneTime, customStringToDate, getErrors,
+  hasDateError, hasTimeError, getClientTimeZone,
+} from '../../utils/DateUtils';
+
+type OnFunction = (value?: number) => void;
+
+export interface DateTimePickerProps{
+  /**
+   * Calendar is opened on input component render
+   */
+  calendarOpened?: boolean,
+  /**
+   * Close calendar on date select
+   */
+  closeOnSelect?: boolean,
+  /**
+   * Date format used by component
+   */
+  dateFormat?: string,
+  /**
+   * Boolean to toggle time display
+   */
+  dateOnly?: boolean,
```

Comment:
What if we have a `timeOnly` option as well? I think there could be some use cases where the dates might be pre-decided and only the time should be selected. I would find it very useful to have this functionality wrapped by this component. Of course this feature could be added later, after this PR is closed, I just thought it would be nice to discuss it.

Reply:
💡